modules/webbrowser/src/webbrowserapp.cpp

```diff
@@ -49,6 +49,8 @@ void WebBrowserApp::OnBeforeCommandLineProcessing(const CefString&,
     commandLine->AppendSwitch("use-mock-keychain");
     commandLine->AppendSwitch("enable-begin-frame-scheduling");
     commandLine->AppendSwitchWithValue("autoplay-policy", "no-user-gesture-required");
+    commandLine->AppendSwitch("--disable-gpu-sandbox");
+    commandLine->AppendSwitch("--no-sandbox");
```

Comment:
Didn't we disable this some time ago @ylvaselling?   Maybe setting these flags is only necessary on Apple?

Reply:
I think we do only need this on apple, it just felt cleaner to not have the ifdef apple in there, happy to add it thou

Reply:
No, I haven't done anything with the sandbox flags. [Here](https://team-openspace.slack.com/archives/CBLM14A87/p1651014724397579) is the message which describes my findings from investigating the CEF flags